tests/jax/integration/executor_test.py

```diff
@@ -13,265 +13,213 @@
+    assert (
+        lambda: x in range(0, num_possible_actions)
+        for x in list(executor._executor.store.actions_info.values())
+    )
 
-    # Change a variable in the policy network
-    parameter_server.set_parameters(
-        {"evaluator_steps": np.full(1, 1234, dtype=np.int32)}
+    assert (
+        lambda: key == "log_prob"
+        for key in executor._executor.store.policies_info.values()
     )
 
-    # Step the executor
-    executor.run_episode()
+    # Observe (without adder)
+    assert not hasattr(executor._executor.store.adder, "add")
 
-    # Check if the executor variable has changed.
-    parameters = executor._executor.store.executor_parameter_client._parameters
-    assert parameters["evaluator_steps"] == 1234
-
-
-#########################################################################
-# Integration test for the executor, adder, data_server, variable_client
-# and variable_server.
-class TestSystemExceptTrainer(System):
-    def design(self) -> Tuple[DesignSpec, Dict]:
-        """Mock system design with zero components.
-
-        Returns:
-            system callback components
-        """
-        components = DesignSpec(
-            **system_init,
-            data_server=OnPolicyDataServer,
-            data_server_adder_signature=ParallelSequenceAdderSignature,
-            extras_spec=ExtrasLogProbSpec,
-            parameter_server=DefaultParameterServer,
-            executor_parameter_client=ExecutorParameterClient,
-            **executor,
-            distributor=Distributor,
-            trainer_parameter_client=mocks.MockTrainerParameterClient,
-            logger=mocks.MockLogger,
-            trainer=mocks.MockTrainer,
-            trainer_dataset=mocks.MockTrainerDataset,
-        )
-        return components, {}
 
+def test_executor_mulit_process_with_adder(test_system_mp: System) -> None:
+    """Test if the executor instantiates processes as expected."""
 
-@pytest.fixture
-def test_system_except_trainer() -> System:
-    """Add description here."""
-    return TestSystemExceptTrainer()
-
-
-# Skip failing test for now
-@pytest.mark.skip
-def test_except_trainer(
-    test_system_except_trainer: System,
-) -> None:
-    """Test if the parameter server instantiates processes as expected."""
-
-    # Environment.
-    environment_factory = functools.partial(
-        debugging_utils.make_environment,
-        env_name="simple_spread",
-        action_space="discrete",
-    )
+    (trainer_node,) = test_system_mp._builder.store.program._program._groups["trainer"]
+    (executor_node,) = test_system_mp._builder.store.program._program._groups[
+        "executor"
+    ]
+    (evaluator_node,) = test_system_mp._builder.store.program._program._groups[
+        "evaluator"
+    ]
+    (parameter_server_node,) = test_system_mp._builder.store.program._program._groups[
+        "parameter_server"
+    ]
+    trainer_node.disable_run()
+    executor_node.disable_run()
+    evaluator_node.disable_run()
```

Comment:
Same here. Can't we just not run the evaluator here? Instead of having to disable it. Or is the idea to test how it integrates with the executor?

Reply:
I think the evaluator is an option for a system to run so if you want me to remove it we can do that!